svc-update/update/simpleUpdate.go

```diff
@@ -294,3 +303,52 @@ func sortTargetList(Targets []string) (map[string][]string, error) {
 	}
 	return returnList, nil
 }
+
+func expandAggregatesCollection(token string, links []string) ([]string, error) {
+	elements := make([]string, 0)
+	for _, link := range links {
+		if strings.Contains(link, "Aggregates") {
+			aggregate, err := getLinksFromAggregate(token, link)
+			if err != nil {
+				return nil, err
+			}
+			elements = append(elements, aggregate...)
+		} else {
+			elements = append(elements, link)
+		}
+	}
+	return elements, nil
+}
+
+func getLinksFromAggregate(token string, aggregate string) ([]string, error) {
```

Comment:
can we change the name of the aggregate variable, so it indicates that it is an URI?

Reply:
Fixed